### PR TITLE
add new block storage

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1959
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/672
https://github.com/open-orchestra/open-orchestra-model-interface/pull/245
https://github.com/open-orchestra/open-orchestra-elastica-bundle/pull/58
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/314
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/62

### CODE REVIEW

File: FrontBundle/Exception/NonExistingAreaException.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace OpenOrchestra\FrontBundle\Exception;
+
+/**
+ * Class NonExistingAreaException
+ */
+class NonExistingAreaException extends \Exception
+{
+
```
Comment (ngilain) [inline FrontBundle/Exception/NonExistingAreaException.php]: @alavieille : empty line to remove

File: FrontBundle/Exception/NonExistingTemplateException.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace OpenOrchestra\FrontBundle\Exception;
+
+/**
+ * Class NonExistingTemplateException
+ */
+class NonExistingTemplateException extends \Exception
+{
+
```
Comment (ngilain) [inline FrontBundle/Exception/NonExistingTemplateException.php]: @alavieille : empty line to remove

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**